### PR TITLE
Aligned diskspace_healthc_check unrequired inputs

### PR DESCRIPTION
**Pull request by tudorlesan**
Signed-off-by: Lesan Tudor <tudor-andrei.lesan@hp.com>

### CODE REVIEW

File: content/io/cloudslang/base/os/linux/diskspace_health_check.sl [hunk 1]
```diff
@@ -32,9 +32,11 @@ flow:
   inputs:
     - docker_host
     - docker_username
-    - docker_password
-    - private_key_file:
+    - docker_password:
         default: "''"
```
Comment (meirwah) [inline content/io/cloudslang/base/os/linux/diskspace_health_check.sl]: @oritstone do we need default: "''"? if we do  required: false ?

Comment (oritstone) [inline content/io/cloudslang/base/os/linux/diskspace_health_check.sl]: We do not need the empty default value. It should be removed. 
And then in the tasks, the input should change to required: false

File: content/io/cloudslang/base/os/linux/diskspace_health_check.sl [hunk 2]
```diff
@@ -32,9 +32,11 @@ flow:
   inputs:
```
Comment (oritstone) [inline content/io/cloudslang/base/os/linux/diskspace_health_check.sl]: In the documentation above, the password & private key file should be documented as optional

**Comment by oritstone** [comment]
Closing. Working on it in #223

**Pull request closed without merging**